sensors/light.py

```diff
@@ -0,0 +1,29 @@
+import time
+import grovepi
+import requests
+import services.telemetry as telemetry
+
+def monitorLight(currentConfig):
+
+    sleepTime = currentConfig["sleep_in_seconds_light"]
```

Comment:
Diese Variable gibt es nicht mehr in der `config.json`, aber da wir ja alle Sensoren gleichzeitig senden lassen wollen, passt es ja hier `sleep_in_seconds` zu nehmen.